test/linear_solver_routines/krylov_helpers/proc_randomized_arnoldi.jl

```diff
@@ -0,0 +1,106 @@
+# Date: 12/04/2024
+# Author: Christian Varner 
+# Purpose: Test the randomized arnoldi iteration
+
+module ProceduralRandomizedArnoldi
+
+using Test, RLinearAlgebra, LinearAlgebra, Random
+
+@testset "Randomized Arnoldi -- Procedural" begin
+
+    Random.seed!(1010)
+
+    # test to make sure it is defined
+    @test isdefined(RLinearAlgebra, :randomized_arnoldi)
+
+    #############################
+    # Error Testing
+    #############################
+    
+    # A is not square
+    nrow = 100
+    sketch_row = 50
```

Comment:
Separate out into another section to indicate that these are test parameters used throughout, or make use of `let` blocks